### ISSUE TITLE
No longer working in latest version of MediaWiki

### ISSUE THREAD

**Issue by SugarD-x** [SimpleMachines/SMF-mediawiki-themes#17]
The smfCurve theme no longer works in the latest version of MediaWiki. (1.32.0 as of this posting). It throws numerous errors, and the page is completely corrupted.

**Comment by jdarwood007** [comment]
We haven't properly tested this for 1.32 or 1.31 for that mater.

**Comment by SugarD-x** [comment]
It definitely doesn't appear to be working on my end in those versions. The page fails to render anything but part of the background, and just throws a spammy error message. A theme I made based on smfCurve also does the same.

**Comment by Antes** [comment]
They probably rewrote whole output template & loading methods yet again... They seems to be changing that every few versions respectively 😆

**Comment by SugarD-x** [comment]
I would not doubt that hahaha...

**Comment by jdarwood007** [comment]
It works ok up to 1.31 it seems.  So most likely they changed the templating engine for the next LTS (1.35).

**Comment by SugarD-x** [comment]
![fgnfgnfgccgxg](https://user-images.githubusercontent.com/4693806/55763839-a2990700-5a1d-11e9-927b-af6a34186c99.PNG)

This is what I get in 1.32.

**Comment by XinYenFon** [comment]
https://github.com/XinYenFon/SMF-mediawiki-themes/archive/1.4.zip test this if you are still interested in 🙂  @SugarD-x

**Comment by SugarD-x** [comment]
I'll try to set up a test wiki ASAP to try this out. :)

**Comment by XinYenFon** [comment]
I ported this theme in 1.34 env. but Curve2 version was working in 1.31+ so this should work in LTS/Latest (but can be broken in some ways).

**Comment by dan4ever** [comment]
Time to update and change function setupSkinUserCss(), its deprecated at 1.32 env.   
I tryed ver 1.4 for MW 1.36.3, not working. 
"The Skin::setupSkinUserCss() method, deprecated in 1.32, was removed. Please use skin registration instead."

**Comment by SugarD-x** [comment]
I apologize I never got back to anyone on this. I don't remember what the issue was, but I ran into some kind of problem back when we were testing that zipped file with MediaWiki that I never got past. I believe it was an environment issue on their end at the time, but I'm not too sure.

I want to say the way they handled skin files was also changed some time around then, which could be a factor in this, and would make sense given @dan4ever's findings.
